tests/rust/manuallydrop.cython.toml

```diff
@@ -0,0 +1,4 @@
+[export]
+exclude = [
```

Comment:
Hmm, yeah, this one seems also a bit unfortunate... I wonder if there's something we could do to make these unnecessary.